sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -425,7 +437,7 @@ Preconditions:
 
 Parameters:
 
-* `dep` - Nodes the created node will be dependent on.
+* `propList` - Zero or more properties can be provided to the constructed node via an instance of property_list.
```

Comment:
Minor comment for here and other identical lines:
*  Would prefer that we line break this so that a line is a maximum 80 chars wide, which I find make the raw document easier to work with.
*  Putting backticks round \`property_list\` would syntax highlight consistently with the main SYCL spec